doc/src/main/asciidoc/inc/_integrations.adoc

```diff
@@ -0,0 +1,46 @@
+
+[[integrations]]
+= Integrations
+
+[[integrations.dekorate]]
+== Dekorate
+
+*fabric8-maven-plugin* provides a Zero Configuration approach to delegate deployment manifests
+to https://github.com/dekorateio/dekorate[Dekorate].
+
+Just by adding a dependency to Dekorate library in the `pom.xml` file, all manifest
+generation will be delegate be delegated to Dekorate.
+
+[source, xml, indent=2]
+----
+<dependencies>
+  <!-- ... -->
+  <dependency>
```

Comment:
Sorry, but I'm not sure if I understand this properly. You say in order to enable Dekorate support we need to add these dependencies in project pom, So how is that different from only adding Dekorate dependencies and not using FMP in the first place?

I thought FMP would include dekorate dependencies and somehow we could turn it's annotation processing on/off depending upon what user configures.

Reply:
Just adding Dekorate dependencies won't enable the use of Dekorate for manifest generation without adding any annotation (Zero-Config) (it will **just for FMP** after this PR https://github.com/dekorateio/dekorate/pull/426 -> but this will use the merge strategy).
This is exactly the point that was discussed on https://bluejeans.com/playback/s/SlaPpPDBTBmlJwZvsXsiHrh7y1n8hJfNvTz2VSg2RBQpHtCaHXywaG87SkcHfGTV where Ioannis presented an example of FMP with Dekorate but an annotation was required for everything to work (`@GeneratorOptions(inputPath = "META-INF/fabric8", outputPath = "META-INF/fabric8")` )

The current approach is simple:
- If no Dekorate dependencies are present, FMP behaves as always (stable and reliable)
- If Dekorate dependencies are present, Dekorate manifest generation is preferred on top of any other Manifest generation strategy (i.e. FMP).
- If Dekorate dependencies are present and -Dfabric8.mergeWithDekorate flag is enabled, manifests will be generated by FMP and then postprocessed+merged by Dekorate (this behavior is unstable)

Just delegating manifest generation to Dekorate is really risky as Dekorate chooses some opinionated defaults too which are incompatible with FMP's, so that's why it's better to leave it to the user.